Modix/IEmoteTypeReader.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Discord;
+using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Modix
+{
+    public class IEmoteTypeReader : TypeReader
+    {
+//TypeReaders have to be asynchronous but we don't do async work, so...
+#pragma warning disable 1998
```

Comment:
Uhh.....no? You absolutely do not need to put the `async` keyword on that method. It simply needs to return `Task<TypeReaderResult>`. Which you can do with `Task.FromResult()`.

Reply:
Gah, totally slipped my mind. Will change